### ISSUE TITLE
Incorrect restriction levels for ascii strings

### ISSUE THREAD

**Issue by sunfishcode** [unicode-rs/unicode-security#25]
unicode-security's `RestrictionLevelDetection` seems to detect an ASCII "a" as `SingleScript` rather than `ASCIIOnly`.

It also seems to detect a Cyrillic "а" as `AsciiOnly` rather than `SingleScript`.

Specifically, this program:

```rust
use unicode_security::restriction_level::RestrictionLevelDetection;

fn main() {
    for string in &[
        "\u{61}",
        "\u{430}",
    ] {
        println!("{} ({}): {:?}",
            string.escape_unicode(),
            string.escape_debug(),
            string.detect_restriction_level(),
        );
    }
}
```

Prints this:
```
\u{61} (a): SingleScript
\u{430} (а): ASCIIOnly
```

**Comment by Manishearth** [comment]
huh, that's exceedingly strange

**Comment by Manishearth** [comment]
https://github.com/unicode-rs/unicode-security/pull/26

fortunately rustc doesn't rely on the distinction being drawn here

**Comment by Manishearth** [comment]
Published as 0.0.6
